pandas/tests/io/excel/test_readers.py

```diff
@@ -1185,6 +1185,13 @@ def test_no_header_with_list_index_col(self, read_ext):
         )
         tm.assert_frame_equal(expected, result)
 
+    def test_one_col_noskip_blank_line(self, read_ext):
+        file_name = "one_col_blank_line" + read_ext
+        data = [0.5, np.nan, 1, 2]
+        expected = DataFrame(data, columns=["numbers"])
+        result = pd.read_excel(file_name)
+        tm.assert_frame_equal(expected, result)
```

Comment:
nit: can you change to `result, expected`

Reply:
:+1: